Content.Client/Research/LatheMenu.cs

```diff
@@ -25,17 +26,22 @@ public class LatheMenu : SS14Window
         private LineEdit AmountLineEdit;
         private LineEdit SearchBar;
         public Button QueueButton;
+        public Button ServerConnectButton;
+        public Button ServerSyncButton;
         protected override Vector2? CustomSize => (300, 450);
 
         public LatheBoundUserInterface Owner { get; set; }
 
         private List<LatheRecipePrototype> _recipes = new List<LatheRecipePrototype>();
         private List<LatheRecipePrototype> _shownRecipes = new List<LatheRecipePrototype>();
 
-        public LatheMenu()
+        public LatheMenu(LatheBoundUserInterface owner = null)
         {
             IoCManager.InjectDependencies(this);
 
+            if (owner != null)
```

Comment:
since `Owner` is `null` by default, this seems to do nothing

Reply:
Whooops. Fixed now.